.github/workflows/docker_image.yml

```
@@ -0,0 +1,87 @@
+---
+name: Docker Image
```

(sorry for this nitpick but consistency)
```suggestion
name: docker_image
```